src/components/BookCard/BookCard.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import styles from './BookCard.module.css'
+import favorite from '../../assets/favorite.svg'
+import book from '../../assets/bookExample.svg'
+
+const BookCard = () => {
+  return (
+    <div className={styles.bookCard}>
+      <div className={styles.cardVisual}>
+        <div className={styles.bookImage}>
+          <img className={styles.image} src={book} alt='book' />
+        </div>
+        <div className={styles.favorite}>
```

Comment:
Create a separate component for this, with one boolean prop that indicates whether the book is in favourites:
`<Favorite inFavorites=true/>` - returns component with filled in heart

You can pass this prop to BookCard component too, so in Homepage component you would have for example:

```
<BookCard inFavorites={true} /> - this book has filled in card
<BookCard inFavorites={false} /> - this book has empty heart
```